tests/integration/api_v2/http/export.test.js

```diff
@@ -16,26 +16,122 @@
+			expect(response).toMap(goodRequestSchema);
+			expect(response.data).toBeInstanceOf(Object);
+			expect(response.data).toMap(exportSchemaAccepted);
+			expect(response.meta).toMap(metaSchema);
+			expect(response.meta).toEqual(expect.objectContaining(expected));
+		});
 
-describe('Account history export API', () => {
-	it('return csv file -> 200 OK', async () => {
-		const validFile = 'test.csv';
-		const response = await api.get(`${endpoint}/${validFile}`, 200);
-		expect(response).toEqual(expectedResponse);
+		it('Schedule file export from account publicKey -> return 202 Accepted', async () => {
+			const expected = { ready: false };
+			const response = await api.get(`${baseUrlV2}/transactions/export?publicKey=${refTransaction2.sender.publicKey}&interval=${startDate}:${endDate}`, 202);
+			expect(response).toMap(goodRequestSchema);
+			expect(response.data).toBeInstanceOf(Object);
+			expect(response.data).toMap(exportSchemaAccepted);
+			expect(response.meta).toMap(metaSchema);
+			expect(response.meta).toEqual(expect.objectContaining(expected));
+		});
 	});
 
-	it('File not exists -> 404 NOT_FOUND', async () => {
-		const invalidFile = 'invalid.csv';
-		const response = await api.get(`${endpoint}/${invalidFile}`, 404);
-		expect(response).toMap(notFoundSchema);
+	describe('File is ready to export', () => {
+		it('scheduled from account address -> return 200 OK', async () => {
+			// Add delay of 10 seconds
+			await new Promise(resolve => setTimeout(resolve, 10000));
+
+			const expected = { ready: true };
+			const response = await api.get(`${baseUrlV2}/transactions/export?address=${refTransaction1.sender.address}&interval=${startDate}:${endDate}`, 200);
+			expect(response).toMap(goodRequestSchema);
+			expect(response.data).toBeInstanceOf(Object);
+			expect(response.data).toMap(exportSchema);
+			expect(response.meta).toMap(metaSchema);
+			expect(response.meta).toEqual(expect.objectContaining(expected));
+		});
+
+		it('scheduled from account from publicKey -> return 200 OK', async () => {
+			// Add delay of 10 seconds
+			await new Promise(resolve => setTimeout(resolve, 10000));
+
+			const expected = { ready: true };
+			const response = await api.get(`${baseUrlV2}/transactions/export?publicKey=${refTransaction2.sender.publicKey}&interval=${startDate}:${endDate}`, 200);
+			expect(response).toMap(goodRequestSchema);
+			expect(response.data).toBeInstanceOf(Object);
+			expect(response.data).toMap(exportSchema);
+			expect(response.meta).toMap(metaSchema);
+			expect(response.meta).toEqual(expect.objectContaining(expected));
+		});
+	});
+
+	describe('Download csv file -> returns 200 OK', () => {
+		it('scheduled from account address -> 200 OK', async () => {
+			const validFileName = `transactions_${refTransaction1.sender.address}_${startDate}_${endDate}.csv`;
+			await api.get(`${baseUrlV2}/exports/${validFileName}`, 200);
```

Comment:
Can you also add the CSV content checks?

Reply:
I have added a check to validate the CSV content.